PLNPluginSchemaMigration.inc.php

```diff
@@ -21,13 +21,6 @@ class PLNPluginSchemaMigration extends Migration {
 	 * @return void
 	 */
 	public function up() {
```

Comment:
Do we need to migrate the existing statuses/data on the new schema?

Reply:
It's not needed, all states will be cleared and retrieved from the PN server.